ui/component/invited/view.jsx

```diff
@@ -0,0 +1,173 @@
+    }
+  }, [referrerSetPending, hasVerifiedEmail]);
+
+  useEffect(() => {
+    if (referrer) {
+      setReferrer(referrer.replace(':', '#'));
+    }
+  }, [referrer]);
+
+  function handleDone() {
+    history.push(`/$/${PAGES.DISCOVER}`);
+  }
+
+  if (referrerSetError === ERRORS.ALREADY_CLAIMED) {
+    return (
+      <Card
+        title={__(`Whoa`)}
+        subtitle={
+          referrerIsChannel
+            ? __(`You've already claimed your referrer, but we've followed this channel for you.`)
+            : __(`You've already claimed your referrer.`)
+        }
+        body={
+          referrerIsChannel && (
+            <div className="claim-preview--channel">
+              <ClaimPreview key={refUri} uri={refUri} actions={''} type={'small'} />
+            </div>
+          )
+        }
+        actions={
+          <div className="card__actions">
+            <Button button="primary" label={__('Done!')} onClick={handleDone} />
+          </div>
+        }
+      />
+    );
+  }
+
+  if (referrerSetError && referredRewardAvailable) {
+    return (
+      <Card
+        title={__(`Welcome!`)}
+        subtitle={__(
+          `Something went wrong with your invite link. You can set and claim your invite reward after signing in.`
+        )}
+        actions={
+          <>
+            <p className="error-text">{__('Not a valid invite')}</p>
+            <div className="card__actions">
+              <Button
+                button="primary"
+                label={hasVerifiedEmail ? __('Verify') : __('Create Account')}
+                navigate={`/$/${PAGES.AUTH}?redirect=/$/${PAGES.REWARDS}`}
+              />
+              <Button button="link" label={__('Explore')} onClick={handleDone} />
+            </div>
+          </>
+        }
+      />
+    );
+  }
+
+  if (!rewardsApproved) {
+    return (
+      <Card
+        title={__(`You're invited!`)}
+        subtitle={
+          referrerIsChannel
+            ? __(
+                `Content freedom and and a present from %channel_name% are waiting for you. Create an account to claim it.`,
+                { channel_name: referrer }
+              )
+            : __(`Content freedom and a present are waiting for you. Create an account to claim it.`)
+        }
+        body={
+          referrerIsChannel && (
+            <div className="claim-preview--channel">
+              <ClaimPreview key={refUri} uri={refUri} actions={''} type={'small'} />
+            </div>
+          )
+        }
+        actions={
+          <div className="card__actions">
+            <Button
+              button="primary"
+              label={hasVerifiedEmail ? __('Finish Account') : __('Create Account')}
+              navigate={`/$/${PAGES.AUTH}?redirect=/$/${PAGES.INVITE}/${referrer}`}
+            />
+            <Button button="link" label={__('Skip')} onClick={handleDone} />
+          </div>
+        }
+      />
+    );
+  }
+
+  return (
+    <Card
+      title={__(`Welcome!`)}
```

Comment:
@seanyesmunt I told @jessopb it was out of scope for his work so not to worry about it, but I think this step is probably the weakest part of the new onboarding/invite system. A brand new user has clicked a link a friend sent them, gone through account creation and email confirmation, and then is kind of just... dumped on the home page (which for now is just them following a single channel).

This doesn't need to be worked out before RC, but it would be good to consider the homepage UX for this case (brand new user following one channel that may be empty or not have much in it)